lib/datadog/tracing/sampling/rule_sampler.rb

```diff
@@ -30,15 +30,20 @@ def initialize(
           default_sampler: nil
         )
           @rules = rules
-          @rate_limiter = if rate_limiter
+          @rate_limiter = if Datadog.configuration.appsec.standalone.enabled
+                            # 1 trace per minute
+                            Core::TokenBucket.new(1.0 / 60, 1.0)
+                          elsif rate_limiter
                             rate_limiter
                           elsif rate_limit
                             Core::TokenBucket.new(rate_limit)
                           else
                             Core::UnlimitedLimiter.new
                           end
-
-          @default_sampler = if default_sampler
+          @default_sampler = if Datadog.configuration.appsec.standalone.enabled
+                               @rules = [SimpleRule.new(sample_rate: 1.0)]
```

Comment:
I think it's quite confusing that we are setting `@rules` when assigning `@default_sampler` value. Should we move this out from here?

Reply:
I agree, I separated rules and default_sampler initializers, thanks!